index.html

```diff
@@ -27,13 +27,19 @@ <h2>TUD Informatik Bachelor VV {{semester}}</h2>
                 <button class="button-show-hidden-courses" ng-if="hiddenCourses().length > 0" class="border-left" ng-click="config.showHiddenCoursesDialog = true">
                     {{hiddenCourses().length}} ausgeblendete Kurse anzeigen
                 </button>
-                <div data-angular-treeview="true" data-tree-id="tree" data-tree-model="data" data-node-id="label" data-node-label="label" data-node-children="children" data-label-template="((node.label)) (( labelForNode(node) ))<span class='credits no-wrap credits-((node.credits))' ng-if='node.credits'>((node.credits)) CP</span>">
+                <div data-angular-treeview="true" data-tree-id="tree" data-tree-model="data" data-node-id="label" data-node-label="label" data-node-children="children" data-label-template="<span class='annotation-((node.annotation))'>((node.label)) (( labelForNode(node) ))</span><span class='credits no-wrap credits-((node.credits))' ng-if='node.credits'>((node.credits)) CP</span>">
                 </div>
             </div>
         </section>
         <aside ng-if="selectedNode" scroll-top-on-change="selectedNode" ng-cloak>
             <div class="close-button button round-button" ng-click="showSelected()">x</div>
             <button ng-click="toggleNodeHiddenStatus(selectedNode, true); showSelected()">Diesen Kurs nicht mehr anzeigen</button>
+
+            <button ng-click="setAnnotation(selectedNode, 1)" class="annotation annotation-1" ng-class="{'annotation-active': selectedNode.annotation === 1}">&nbsp;</button>
```

Comment:
The following lines are a little repetitious - can you maybe put the annotations (in this case `[1, 2, 3, 4, 5]`) in the `$scope` and repeat via `ng-repeat` over them (= generating the buttons dynamically).